Mollier/SAM.Core.Mollier.UI/Modify/AddMollierPoints.cs

```diff
@@ -9,22 +10,89 @@ public static partial class Modify
 
         public static Series AddMollierPoints(this Chart chart, IEnumerable<UIMollierPoint> uIMollierPoints, MollierControlSettings mollierControlSettings)
```

Comment:
Return Series -> List
Move to SAM.Core.Mollier.Create
Rename to Series. Example below:

namespace SAM.Core.Mollier.UI
{
public static partial class Create
{
public static List Series(this Chart chart, IEnumerable<UIMollierPoint> uIMollierPoints, MollierControlSettings mollierControlSettings)
{
//Add code here
}
}
}